sillsdev/web-languageforge
# migrate qa naming to staging instead

## Description

This PR utilizes a different name for our new staging environment

## Checklist


## QA testing

Testers, use the following instructions on [qa.languageforge.org](https://qa.languageforge.org). Post your findings as a comment and include any meaningful screenshots, etc.

- Ensure the header is present and valued properly on all responses `X-Robots-Tag noindex`
